### ISSUE TITLE
componentWillReceiveProps called on initial render

### ISSUE THREAD

**Issue by c58** [developit/preact#60]
In react `componentWillReceiveProps` called only on re-render from parent. In Preact it is called on initial render too. It is a problem for some React components (react-router for example)

**Comment by developit** [comment]
Which version of Preact? I will check again but this shouldn't be the case in 3.x.

**Comment by c58** [comment]
3.4.0
```javascript
class App extends Component {
	componentWillReceiveProps(props) {
		console.log('there');
	}
	render() {
		return (
			<div>Hi</div>
		);
	}
}
render(<App />, document.body);
```

**Comment by developit** [comment]
Ah okay I must have fixed this in my 4.0 work.
